packages/view/backend/sirius-components-view-form/src/main/resources/model/form.ecore

```diff
@@ -11,6 +11,12 @@
     <eLiterals name="LIST"/>
     <eLiterals name="TOGGLEABLE_AREAS" value="1"/>
   </eClassifiers>
+  <eClassifiers xsi:type="ecore:EEnum" name="LabelPlacement">
+    <eLiterals name="end"/>
+    <eLiterals name="top" value="1"/>
+    <eLiterals name="start" value="2"/>
+    <eLiterals name="bottom" value="3"/>
```

Comment:
Not sure about "start" and "end". Maybe "left" & "right" would be more explicit?

Reply:
Well apparently its the names used by MUI's FormControlLabel, so let it be.
I guess they handle RTL text?

Reply:
Yes, we choose these labels to be consistent with MUI's FormControlLabel.